meteor/server/api/ingest/bucketAdlibs.ts

```diff
@@ -29,7 +29,12 @@ export function updateBucketAdlibFromIngestData(
 		studio,
 		showStyle._id,
 		showStyle.showStyleVariantId,
-		new NotesContext('Bucket Ad-Lib', 'bucket-adlib', false)
+		new NotesContext(
+			'Bucket Ad-Lib',
+			'bucket-adlib',
+			false,
+			[blueprintId, studio.blueprintId].map(unprotectString).filter((id): id is string => id !== undefined)
```

Comment:
What's this about? You can use `unprotectStringArray` if you need to pass some protectedStrings array as a string array.

Reply:
That's a good tip, thanks. I'll push an update 👍

Reply:
I was just hoping that there's a way of not running the map and filter. I don't think either `blueprintId` or `studio.blueprintId` can be undefined (as per typings), so just a simple overload should be fine.

Reply:
this has been changed in #336, which will be merged in at the same time, so isnt worth worrying about